lab7.md

```diff
@@ -82,7 +82,144 @@ Each time you create an issue the count will increase. You can see the payload s
 
 ## Analyse new Issues on GitHub
 
+
+### Deploy Sentiment Analysis function
+
+In order to use this issue-bot function, you will need to deploy the Sentiment Analysis function first.
+This is a python function that provides a rating on sentiment positive/negative (polarity -1.0-1.0) and subjectivity provided to each of the sentences sent in via the TextBlob project.
+
+You can checkout the function from [faas/sample-functions](https://github.com/openfaas/faas/tree/master/sample-functions).
+
+Deploy with: 
+
+```
+curl -s http://fuh83fhfj.ngrok.io/system/functions --data-binary \
```

Comment:
this should use localhost:8080 and please use the native CLI for the deployment or the UI (it's available in the store)